OpenDreamRuntime/Objects/DreamList.cs

```diff
@@ -414,4 +414,43 @@ public sealed class DreamFilterList : DreamList {
             return appearance;
         }
     }
+
+    // world.contents list
+    // Operates on a list of all atoms
+    public sealed class WorldContentsList : DreamList {
```

Comment:
won't this still allocate a unused list due to inheriting dreamlist?

Reply:
It would. So do all the other special lists, it would be a good idea in the future to make DreamList into an abstract class or something similar.